README.md

```diff
@@ -115,9 +117,7 @@ Using jQuery, add an event listener for clicks on the button with the id
 body, that says "hello".
 
 Your Answer:
-```js
-// your code here
-```
+$ .on(click[,alert][,hello],<p>)
 
```

Comment:
Nope. The general structure of `$`and `.on()` is correct, but this is not valid js or jQuery. The biggest thing: There's no function! Take a look at the solution below, and please review the jQuery lesson notes: https://github.com/ga-dc/curriculum/tree/master/02-intermediate-front-end/jquery and/or take a look at the CodeAcademy jQuery course : https://codecademy.com/learn/jquery

```js
$('#greeting').on("click", function() {
  $('body').append("<p>hello</p>");
});
```